src/main/rx/fport.c

```diff
@@ -25,7 +25,7 @@
 
 #include "platform.h"
 
-#if defined(USE_SERIALRX_FPORT)
+#if defined(USE_SERIAL_RX) && defined(USE_SERIALRX_FPORT)
```

Comment:
Surprised that you found this to make a difference - this is effectively no change as `USE_SERIALRX_FPORT` will never be defined if `USE_SERIAL_RX` isn't.

Reply:
The saving value shows difference between this line being effective or not.

Reply:
I see.